dom.bs

```diff
@@ -2724,25 +2783,26 @@ before a <var>child</var>, with an optional <i>suppress observers flag</i>, run
     <p>For each <a>shadow-including inclusive descendant</a> <var>inclusiveDescendant</var> of
     <var>node</var>, in <a>shadow-including tree order</a>:
 
-    <ol>
-     <li><p>Run the <a>insertion steps</a> with <var>inclusiveDescendant</var>.
+     <ol>
```

Comment:
This is also wrong.

Reply:
Because this snippit is outdated I can't quite tell if it is still relevant, but I think it is fixed. Can you take a look?